11/umbraco-forms/developer/magic-strings.md

```diff
@@ -82,15 +82,16 @@ For example, to truncate a string value read from an Umbraco page field with ali
 
 Umbraco Forms ships with the following filters:
 
-| Filter                        | Function   | Arguments            | Example                           |
-| ----------------------------- | ---------- | -------------------- | --------------------------------- |
-| Convert string to lower case  | `lower`    |                      | `[#field lower]`                  |
-| Convert string to upper case  | `upper`    |                      | `[#field upper]`                  |
-| Truncate a string             | `truncate` | number of characters | `[#field truncate: 10]`           |
-| Format a number               | `number`   | format string        | `[#field number: #0.##%]`         |
-| Format a number as a currency | `currency` |                      | `[#field currency]`               |
-| Format a date                 | `date`     | format string        | `[#field date: dd-MM-yyyy HH:mm]` |
-| Bound a number                | `bound`    | min and max bound    | `[#field bound: 1: 10]`           |
+| Filter                        | Function   | Arguments            | Example                             |
```

Comment:
The examples are not rendered properly on the doc site. Can we try without ``?

You can look at it here:
https://app.gitbook.com/o/vHdmkfI8smZW50A5yIZD/s/2XErz7PeSuiFTiCbGF0w/~/revisions/bpJ1KS92dCT64kLpkGIB/developer/magic-strings

Reply:
What's the issue you see here please?  Other than the text wrapping, which we can't do too much about, it looks OK to me.

Reply:
When I previewed the file in GitBook, the example column showed me half examples and it wasn't being rendered correctly. I took a look at the current article now on the docs site and looks like its being rendered correctly, so nothing to be done here.